main/guides/orchestration/dapp-orchestration-basics/installation/index.md

```diff
@@ -106,6 +109,19 @@ If you need to restart your environment for any reason, use the following comman
 make teardown ; make stop; make stop-forward; make clean; make; make port-forward
 ```
 
+## Troubleshooting Docker
+During setting up the environment, you may run into a few issues with docker. Here are two common setup requirements.
+
+### Disable Kubernetes
```

Comment:
I thought we have to enable it, not disable it. That's what I had to do.

Reply:
Also, we should recommend somewhere to click the "reset k8s cluster" button before spinning it up